app/controllers/graphql_controller.rb

```diff
@@ -21,6 +24,13 @@ def current_user
     credential.current_user_id
   end
 
+  def admin_status
+    credential = setup_credentials
```

Comment:
This is duplicated code across your two context methods. I'd look to extract the setup and testing of ok for the credential to a `before_filter` see https://guides.rubyonrails.org/action_controller_overview.html#filters to avoid double loading the bearer token from the auth header. 
```
    before_filter :execute :setup_and_validate_credential
```